gpswearos/build.gradle

```diff
@@ -0,0 +1,68 @@
+plugins {
+    id 'com.android.application'
+    id 'org.jetbrains.kotlin.android'
+}
+
+android {
+    namespace 'com.android.gpswearos'
+    compileSdk 32
+
+    defaultConfig {
+        applicationId "com.android.gpswearos"
+        minSdk 30
+        targetSdk 32
+        versionCode 1
+        versionName "1.0"
+        vectorDrawables {
+            useSupportLibrary true
+        }
+
+    }
+    lint {
+        disable "JvmStaticProvidesInObjectDetector", "FieldSiteTargetOnQualifierAnnotation", "ModuleCompanionObjects", "ModuleCompanionObjectsNotInModuleParent"
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
```

Comment:
`minifyEnabled` should be `true` for release builds.

Reply:
> `minifyEnabled` should be `true` for release builds.

Thanks, I will replace it.